app/styleguides/_default/components/header.css

```diff
@@ -1,77 +1,66 @@
 @import '_default/common/_vars.css';
 
-.header {
-  display: flex;
-  flex-wrap: wrap;
-  justify-content: space-between;
-  margin: 9px 0 0 8px;
+header.header {
+  margin: 0 auto;
+  max-width: 100%;
 
-  .menu-wrapper {
-    height: 46px;
-    margin: 20px 0 0;
-    width: 46px;
+  .header-inner {
+    text-align: center;
+  }
 
-    /* put in a pretend menu placeholder for now */
-    .menu {
-      border: 2px solid black;
-      border-radius: 50%;
-      display: block;
-      height: 38px;
-      margin: auto;
-      width: 38px;
-    }
+  .header__logo-link:focus {
+    outline: none;
   }
 
-  .heading-wrapper {
-    align-items: center;
-    display: flex;
-    flex-grow: 1;
-    text-align: center;
+  @media screen and (min-width: $lg) {
+    margin: 27px auto 20px;
+    max-width: $wrapperMaxWidth;
+  }
+}
 
-    .logo {
-      height: max-content;
-    }
 
-    .title-logo {
-      margin: 0;
-    }
+.header__tagline {
+  font-family: $sans-serif-stack;
+  margin: 15px 0;
+}
 
-    .title-logo svg {
-      width: max-content;
-    }
+.header__logo-link,
+.header__title-logo{
+  display: inline-block;
+  svg, .header__logo {
+    max-width: 100%;
+    width: max-content;
   }
 }
 
-@media screen and (min-width: 1180px) {
-  .header {
-    justify-content: flex-start;
-    margin: 27px auto 20px;
-    max-width: $wrapperMaxWidth;
-
-    .menu-wrapper {
-      height: 43px;
-      width: 43px;
+.header__social-icons {
+  align-items: center;
+  display: flex;
+  justify-content: center;
 
-      /* put in a pretend menu placeholder for now */
-      .menu {
-        border: 2px solid black;
-        border-radius: 50%;
-        display: block;
-        height: 43px;
-        margin: auto;
-        width: 43px;
-      }
+  svg {
+    height: auto;
+    width: 26px;
+  }
+  .share-link {
```

Comment:
Please add a new line above this one to separate css blocks

Reply:
👍